cyipopt/scipy_interface.py

```diff
@@ -129,7 +129,7 @@ def __init__(self,
 
         if hess is not None:
             self.obj_hess = hess
-        if jac is None:
+        if not jac:
```

Comment:
When `jac` is `False`, the code currently treats it as though it were a callable.